apps/teamsjs-diagnostic-app/src/index.tsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { createRoot } from "react-dom/client";
+import App from "./components/App";
+import "./index.css";
+
+const container = document.getElementById("root");
+const root = createRoot(container!);
```

Comment:
Can you explain me why we need `!` non-null operator? what if root is null?

Reply:
When I remove the !  i get the following:

Argument of type 'HTMLElement | null' is not assignable to parameter of type 'Container'.
  Type 'null' is not assignable to type 'Container'.ts(2345)
const container: HTMLElement | null

Reply:
What are your thoughts on the below code:

```
const container = document.getElementById("root");
if (container) {
  const root = createRoot(container);
  // Proceed with further initialization
} else {
  console.error("Root container not found");
}
```

Reply:
That works, i will change it to that!